### PR TITLE
README.md - Add TOC and "Enable mgr dashboard" section

### PR DESCRIPTION
**Pull request by guymguym**

### CODE REVIEW

**Comment by guymguym** [comment]
Hey @leseb,
Let me know if anything is missing here for merging it.
Regards

**Comment by leseb** [comment]
@guymguym oops sorry. Well the thing that bugs me is the rebuilding of the image itself, making changes in the image and then re-using it. To me, this sounds more like a workaround to something that should be fixed inside the container image from the get go.

So I'd be in favor of highlighting this as a temporarly fix, thanks.

**Comment by guymguym** [comment]
I agree, I would also expect it to work without workarounds.

File: README.md [hunk 1]
```diff
@@ -202,3 +214,69 @@ $ ./cn cluster start mycluster -i mimic
 ```
 
 It is also possible to create new aliases as detailed [here](CONFIGURATION.md)
+
+## Enable mgr dashboard
```
Comment (leseb) [inline README.md:219]: Please mark this as a temporary hack.

**Review by leseb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (guymguym) [reply README.md:219]: done

**Review by leseb — APPROVED** [review]

**Pull request merged**